common/system/tasks/radio_manager.cpp

```diff
@@ -292,14 +314,19 @@ void radioManager_transmitString(int radioId, uint8_t *data, size_t len) {
       }
     }
 #else
-    radioManager_sendInternal(radioId);
+    sendInternal(transmitPacket);
 #endif
 
     len -= txLen;
     data += txLen;
   }
 }
 
-void radioManager_transmitStringDefault(uint8_t *data, size_t len) {
-  radioManager_transmitString(RADIO_CLI_ID, data, len);
+RadioManager radioManagers[NUM_RADIO];
```

Comment:
Generally speaking, I think it would be ideal to have a singular radio manager that manages multiple radios and takes in a radio ID for each. Having a different radio manager for each radio doesn't seem particularly intuitive from a high level perspective; I'd expect to have *the* radio manager if I had a reference to one and so I'd be able to use all of the radios.